src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir-Book", "Avenir-Heavy", sans-serif;
+}
+
+form {
+  margin-bottom: 20px;
+  margin-top: 20px;
+  position: relative;
+}
+
+input {
+  outline: none;
+}
+
+.bigSearch-svg {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.bigSearch {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.smallSearch-svg {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.smallSearch {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+/* style properies */
+
+.bigSearch,
+.smallSearch {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir-Heavy", "Avenir-Book", sans-serif;
```

Comment:
Why do you need 2 different font families here? "Avenir-Heavy", "Avenir-Book"

Reply:
Additional, if first doesn't work